package.json

```diff
@@ -57,6 +57,7 @@
     "lodash": "4.17.21",
     "luxon": "2.5.2",
     "mysql": "2.18.1",
```

Comment:
@kschelonka Should we remove this dependency?

Reply:
ah thank you, I thought I uninstalled already. nice catch

Reply:
I also removed the otel library for it